lib/jekyll-mentions/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module JekyllMentions
-  VERSION = "1.2.0"
+  VERSION = "1.2.0".freeze
```

Comment:
This shouldn't be necessary because of `frozen_string_literal: true`

Reply:
yeah but Rubcop complains for older Rubies and we don't want to target Ruby 2.3+ for now 😉

Reply:
Then let's ignore this cop. This is nonsensical.

Reply:
I guess it's what Jekyll does, so 🤷‍♂️

Reply:
I agree, we should fix that in Jekyll's config.

Reply:
`frozen_string_literal: true` was introduced in Ruby 2.3.. So it makes sense that older Rubies won't have a frozen value assigned to the constant

Reply:
Why do both?

Reply:
By adding an explicit `freeze` call, the constant is guaranteed frozen on all supported Rubies instead of just 2.3+.

Reply:
So let’s drop the magic comment.

Reply:
:) why is this bothering you so much..?

Reply:
I’m grumpy 😜
You’re right; it doesn’t matter.

Reply:
Don't worry @pathawks , I promise we'll get rid of all those useless `.freeze` at the end of march when we stop supporting EOL rubies :)